syntax = "proto3";
package rdma;

message HelloMessage {
    string name = 1;
}
